ros_buildfarm/pulp.py

```diff
@@ -140,6 +141,10 @@ def enumerate_pkgs_in_repo_ver(self, repo_ver_href):
         return PulpPageIterator(
             self._rpm_packages_api.list, repository_version=repo_ver_href)
 
+    def enumerate_remotes(self):
+        return PulpPageIterator(
+            self._rpm_remotes_api.list)
```

Comment:
Nitpick: could be on the same line, no need to wrap.

Reply:
1c6a496